### DISCUSSION TITLE
How to add new strategy?

### DISCUSSION THREAD

**Discussion by ghost** [stevearc/overseer.nvim#90] [Q&A] [answered]
Hello, I  wanna ask if it's the correct way to add support for new strategy by implementing the methods listed here [terminal](https://github.com/stevearc/overseer.nvim/blob/master/lua/overseer/strategy/terminal.lua) and here [toggleterm](https://github.com/stevearc/overseer.nvim/blob/master/lua/overseer/strategy/toggleterm.lua), like `ToggleTermStrategy:reset` etc.. I am using `FTerm`, I guess it's pretty similar and easy to add it.

**Comment by stevearc** [comment, marked as answer]
You got it! The toggleterm strategy will probably be pretty similar, but any of them should be a decent guide.
